assets/svelte/components/SelectedElementFloatingMenu/DragMenuOption.svelte

```diff
@@ -58,10 +57,9 @@
   }
 
   function updateHandleCoords(currentRect: LocationInfo, isParent: boolean) {
-    relativeWrapperRect = document
-      .getElementById("ui-builder-app-container")
-      .closest(".relative")
-      .getBoundingClientRect()
+    let appContainer = document.getElementById("ui-builder-app-container")
+    if (!appContainer) return
+    relativeWrapperRect = appContainer.closest(".relative").getBoundingClientRect()
```

Comment:
This is unrelated with the feature but I noticed that when toggling between the code and visual editor, sometimes this code runs befose the app's container is attached to the DOM.